modules/integration/src/test/scala/zio/elasticsearch/HttpExecutorSpec.scala

```diff
@@ -2771,6 +2772,69 @@ object HttpExecutorSpec extends IntegrationSpec {
             }
           } @@ after(Executor.execute(ElasticRequest.deleteIndex(geoPolygonIndex)).orDie)
         ),
+        suite("intervals query")(
+          test("intervalMatch returns only matching document") {
+            checkOnce(genDocumentId, genTestDocument, Gen.alphaNumericString, genDocumentId, genTestDocument) {
+              (idMatch, docMatch, targetWord, idNoMatch, docNoMatch) =>
+                val docShouldMatch    = docMatch.copy(stringField = s"prefix $targetWord suffix")
+                val docShouldNotMatch = docNoMatch.copy(stringField = "completely unrelated text")
+                val field             = TestDocument.stringField.toString
+                val query             = intervals(field, intervalMatch(targetWord))
```

Comment:
Please use `TestDocument.stringField`. And I guess no need for `field` as a separate val.